### ISSUE TITLE
Output from wtc changed due to updated check.data

### ISSUE THREAD

**Issue by vsimko** [tgouhier/biwavelet#41]
**I'm not sure if this is a bug or a bugfix**

The commit 820979b0c2c1d7500bf14247883bcf4df263b172 changed the output from `wtc` function after the `check.data` function has been updated:

The following code demonstrates it:

``` r
data(enviro.data)
MEI <- subset(enviro.data, select = c("date", "mei"))
NPGO <- subset(enviro.data, select = c("date", "npgo"))
wtc.mei.npgo <- wtc(MEI, NPGO, nrands = 10)
par(oma = c(0, 0, 0, 1), mar = c(5, 4, 4, 5) + 0.1)
plot(wtc.mei.npgo, plot.cb = TRUE, plot.phase = TRUE)
```

Original image from `wtc` still in commit:
- b99b24ea249311f14be9cb97b6a127c27830a5b9
  Merge pull request #34 from vsimko/master
  parameters for sqrt() explicitly casted to double

![image](https://cloud.githubusercontent.com/assets/7080773/15087592/784d7430-13ea-11e6-9f1c-bbbb8fc92e5d.png)

New image from `wtc` introduced in commits:
- 820979b0c2c1d7500bf14247883bcf4df263b172
  > Fix check for consistent time steps.
- efcb01a4d30d18e20f23cfd94ad61ebb39b013d0
  > Merge branch 'master' of github.com:tgouhier/biwavelet
- bd796506314e8ff9ab8b6856bcec899a0ffa1ed8
  > Fix check for consistent time steps.
- 9cc0b29b9651c870044965b73600fec27bc70268
  > Convert time to time index for plotting.

![image](https://cloud.githubusercontent.com/assets/7080773/15087642/ecf1435c-13ea-11e6-8e89-39a7452e1073.png)

**Comment by vsimko** [comment]
@tgouhier I'm not sure if this is a bug or a bugfix

**Comment by tgouhier** [comment]
It's a silly bug in check.datum that I fixed in version 0.20.2. Could you add the changes that you made to the NEWS.Rd file so that I may release this update?
